src/TuDa.CIMS.Web/Components/Layout/MainLayout.razor

```diff
@@ -2,7 +2,7 @@
 
 <div class="page">
     <div class="sidebar">
-        <NavMenu/>
+        <NavBar/>
     </div>
 
     <main>
```

Comment:
Currently, the `NavBar` is shown in a sidebar. To use it as a real `NavBar` you should remove the div around it and move it into main.
```
<main>
    <NavBar/>
    <article class="content px-4">
        @Body
    </article>
</main>
```